gitfs/worker/sync.py

```diff
@@ -87,7 +87,9 @@ def on_idle(self):
                 self.commit(self.commits)
                 self.commits = []
             log.debug("Start syncing")
-            self.sync()
+            while not self.sync():
```

Comment:
should this have some sort of back-off mechanism? sleep for a while maybe? right now this has the potential to become a cpu-hogging busy loop